codec-http2/src/main/java/io/netty/handler/codec/http2/HttpConversionUtil.java

```diff
@@ -620,7 +622,16 @@ private static void setHttp2Scheme(HttpHeaders in, URI uri, Http2Headers out) {
             translations = request ? REQUEST_HEADER_TRANSLATIONS : RESPONSE_HEADER_TRANSLATIONS;
         }
 
-        public void translate(Entry<CharSequence, CharSequence> entry) throws Http2Exception {
+        public void translateHeaders(Iterable<Entry<CharSequence, CharSequence>> inputHeaders) throws Http2Exception {
+            for (Entry<CharSequence, CharSequence> entry : inputHeaders) {
+                translateHeader(entry);
+            }
+            if (cookies != null) {
+                output.add(COOKIE, cookies.toString());
+            }
+        }
+
+        private void translateHeader(Entry<CharSequence, CharSequence> entry) throws Http2Exception {
```

Comment:
@kevinoliver sorry for nit-picking but if you would fold this method into `translateHeaders` you could just have the `StringBuilder` as local variable and remove the declaration of `private StringBuilder cookies`. IMHO this would be preferable.

Reply:
nope, not a nitpick. will fix it up.